core/src/main/antlr/xtdb/antlr/Sql.g4

```diff
@@ -595,11 +595,11 @@ tableTimePeriodSpecification
 
 // replace usages of this with dateTimeExpr when possible - #3306
 periodSpecificationExpr
-    : literal #PeriodSpecLiteral
-    | parameterSpecification #PeriodSpecParam
-    | ('NOW' | 'CURRENT_TIMESTAMP') #PeriodSpecNow
+    : ('NOW' | 'CURRENT_TIMESTAMP') #PeriodSpecNow
+    | expr #PeriodSpecExpr
```

Comment:
possible to remove `periodSpecificationExpr` entirely now (i.e. inline it as `expr`)?